pkg/test/framework/features/allowlist.txt

```diff
@@ -1514,6 +1515,7 @@ telemetry_outboundtrafficpolicy,TestOutboundTrafficPolicy_AllowAny
 telemetry_outboundtrafficpolicy,TestOutboundTrafficPolicy_RegistryOnly
 telemetry_requestclassification,TestRequestClassification
 telemetry_stackdriver,TestStackdriverMonitoring
+telemetry_stackdriver,TestStackdriverHTTPAuditLogging
```

Comment:
I think this should be resolved by adding `.Features(...)` on the test instead of allowlisting it

Reply:
Do you think this is correct https://github.com/istio/istio/pull/28680/files#diff-012ed15b784ec47d12f52191781b2ce89c73d01996c592067bbeef9b3d4d250eR47 ? is there any other config required for the Features label?